ansible_base/lib/serializers/common.py

```diff
@@ -98,3 +101,15 @@ class Meta(CommonModelSerializer.Meta):
 class ImmutableCommonModelSerializer(AbstractCommonModelSerializer):
     class Meta(AbstractCommonModelSerializer.Meta):
         fields = AbstractCommonModelSerializer.Meta.fields + ['created', 'created_by']
+
+
+class UneditableSystemUserSerializer(CommonModelSerializer):
```

Comment:
I would be inclined to move the `validate_is_superuser` method here as well, and have it be a generic DAB user serializer.

Reply:
Sounds reasonable to me.

Reply:
OK, did that, please take a look.